ISISComputingGroup/EPICS-inst_servers
# Ticket2559: Create configurations for forwarder from blockserver

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/2559

### To test

* Start epics to kafka forwarder
* Start the BlockServerToKafka, pointing it at the correct data and config topics and supplying an instrument prefix with a running blockserver
* Confirm PV data is being put into the data topic based on the blocks
* Change config
* Confirm data for new blocks is in data topic

---

#### Code Review


### Functional Tests


### Final steps
